src/pages/testInMay/testInMay.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+
+ const showWorker = (a) => {
+    
+    console.info(a)
+ 
+    return false;
+}
+
+class Worker {
```

Comment:
replace it class in own file placed in src/utils/helpers. And import it in this file